.travis.yml

```diff
@@ -1,41 +1,40 @@
 services:
-  - docker
-
+- docker
 branches:
   only:
   - exercise-8
-
+  - exercise-9
 script:
-  - #docker build --tag todo-app-dev --target development .
-  - #docker build --tag todo-app-test --target test .
-  - docker build --tag $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT --target production  .
-  - #docker run -e MY_SECRET -e SERVER_TOKEN -e DEVELOPER_API_KEY -e FLASK_APP -e FLASK_ENV todo-app-test:latest
-  - echo "Push Production image to Docker Hub"
-  - echo $DOCKER_PWD
-  - echo $DOCKER_USER
-  - echo $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT
-  - echo $DOCKER_PWD | docker login -u "$DOCKER_USER" --password-stdin
-  - docker push $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT
-
-  - echo "Push the TODO list to Heroku"
-  - docker pull $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT
-  - docker tag $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT registry.heroku.com/sixela-app/web
-  - # heroku container:login
-  - echo $HEROKU_API_KEY | docker login --username=_ --password-stdin registry.heroku.com 
-  - docker push registry.heroku.com/sixela-app/web
-  - heroku container:release web -a sixela-app
-
-env:  
+- docker build --tag $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT --target production  .
+-
+- echo "Push Production image to Docker Hub"
+- echo $DOCKER_PWD
```

Comment:
It would be good to get your tests back in, particularly as you've already made the effort to fix them up for the Mongo changes